### PR TITLE
Updated MacOS install note & formatting

### PR DESCRIPTION
**Pull request by davrax**
Added to the install note for MacOS on the M1, as Homebrew changed their default install directory for Apple Silicon (paths referenced further below are from an x64 installation). Also demoted the old version paragraphs to paragraph 3, nested under the "Previous Version" paragraph 2.

### CODE REVIEW

**Comment by PRMerger4** [comment]
@davrax : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by AnnaMHuff** [comment]
@David-Engel  - Can you review the proposed changes?

*IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.

Thanks!

#label:"aq-pr-triaged"

File: docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md [hunk 1]
```diff
@@ -20,8 +20,9 @@ This article explains how to install the Microsoft ODBC Driver for SQL Server on
...
 
-> [!Note]
-> The Microsoft ODBC driver for SQL Server on macOS is only supported on the x64 architecture through version 17.7. Apple M1 (ARM64) support was added starting with version 17.8. The architecture will be detected and the correct package will be automatically installed by the Homebrew formula. If your command prompt is running in x64 emulation mode on the M1, the x64 package will be installed. If you're not running in emulation mode in your command prompt, the ARM64 package will be installed.
+> :warning: Note
+> The Microsoft ODBC driver for SQL Server on macOS supports the x64 architecture through version 17.7. In version 17.8, Apple Silicon (ARM64) support was added, and will be the only architecture to receive future support. The architecture will be detected and the correct package will be automatically installed by the Homebrew formula. If your terminal is running in x64 emulation mode on the M1, the x64 package will be installed. If you're not running in emulation mode in your terminal, the ARM64 package will be installed.
```
Comment (David-Engel) [inline docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md:24]: > In version 17.8, Apple Silicon (ARM64) support was added, and will be the only architecture to receive future support.

This is incorrect. Both architectures are currently supported and no decision has been made to stop ODBC driver support for x64.

File: docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md [hunk 2]
```diff
@@ -38,7 +39,7 @@ HOMEBREW_NO_ENV_FILTERING=1 ACCEPT_EULA=Y brew install msodbcsql18 mssql-tools18
 
 The following sections provide instructions for installing previous versions of the Microsoft ODBC driver on macOS.
 
-## <a id="17"></a> Microsoft ODBC 17
+### <a id="17"></a> Microsoft ODBC 17
```
Comment (David-Engel) [inline docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md:42]: Revert. We want the two previous versions to be available via the right-side context links.

File: docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md [hunk 3]
```diff
@@ -53,7 +54,7 @@ HOMEBREW_NO_ENV_FILTERING=1 ACCEPT_EULA=Y brew install msodbcsql17 mssql-tools
 > If you installed the v17 `msodbcsql` package that was briefly available, you should remove it before installing the `msodbcsql17` package. This will avoid conflicts. The `msodbcsql17` package can be installed side by side with the `msodbcsql` v13 package.
 
 
-## <a id="13.1"></a> ODBC 13.1
+### <a id="13.1"></a> ODBC 13.1
```
Comment (David-Engel) [inline docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md:57]: Revert. We want the two previous versions to be available via the right-side context links.

File: docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md [hunk 4]
```diff
@@ -20,8 +20,9 @@ This article explains how to install the Microsoft ODBC Driver for SQL Server on
...
+> :warning: Note
+> The Microsoft ODBC driver for SQL Server on macOS supports the x64 architecture through version 17.7. In version 17.8, Apple Silicon (ARM64) support was added, and will be the only architecture to receive future support. The architecture will be detected and the correct package will be automatically installed by the Homebrew formula. If your terminal is running in x64 emulation mode on the M1, the x64 package will be installed. If you're not running in emulation mode in your terminal, the ARM64 package will be installed. 
+> Additionally, the Homebrew default directory changed with the M1, to ```/opt/homebrew```. The paths below use the x64 Homebrew paths, which default to ```/usr/local```, so your filepaths will vary accordingly.
```
Comment (David-Engel) [inline docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md:25]:
```suggestion
> Additionally, the Homebrew default directory changed with the M1, to `/opt/homebrew`. The paths below use the x64 Homebrew paths, which default to `/usr/local`, so your file paths will vary accordingly.
```

**Review by David-Engel — CHANGES_REQUESTED** [review]
I'm fine with adding line 25 (with minor edits). The rest of the changed lines should be reverted.

**Comment by PRMerger15** [comment]
@davrax : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@davrax : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by David-Engel — APPROVED** [review]
Looks good to me.

**Comment by David-Engel** [comment]
#sign-off

**Comment by jborsecnik** [comment]
Thanks. Due to the upcoming Build conference, all merging and publishing is on hold until the morning of May 24.
This PR is ready for merge on 5/24.

**Pull request merged**